tools/rst-lint.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# run rst-lint (reStructuredText linter) with preferred settings
+
+set -eu
+
+cd "$(dirname -- "${0}")/.."
+
```

Comment:
This is good information.  Would you mind putting this line:
```sh
rst-lint --level info README.rst
```
into the readme itself?  You could add information about runnning mypy. pflake8, etc. there too?  Maybe create a new section?

Reply:
Yes, sounds great.  Would you mind adding this line to that section?

Reply:
Fixed.